src/sentry/api/serializers/models/dashboard.py

```diff
@@ -145,8 +145,10 @@ def get_attrs(self, item_list, user):
         return result
 
     def serialize(self, obj, attrs, user, **kwargs):
+        from sentry.api.serializers.rest_framework.base import camel_to_snake_case
```

Comment:
Are we unable to import at the top of the file?

Reply:
I was getting an import error when trying to import at the top of the file:
```
INTERNALERROR>     from sentry.api.serializers.rest_framework.base import camel_to_snake_case
...
INTERNALERROR>     from sentry.plugins.base import plugins
INTERNALERROR> ImportError: cannot import name 'plugins' from partially initialized module 'sentry.plugins.base' (most likely due to a circular import) (/Users/nsaynorath/workspace/sentry/src/sentry/plugins/base/__init__.py)
```

Maybe one thing I can try is moving this helper out into the helpers directory, so it's not doing all of the `sentry/api/serializers/rest_framework/__init__.py` imports, but it looks like all the code for the CamelSnakeSerializer and CamelSnakeModelSerializer lives in there now.

Reply:
Nah if there's an issue at the top, this is fine :+1: